cadc-util-fs/src/main/java/org/opencadc/util/fs/ExtendedFileAttributes.java

```diff
@@ -78,6 +78,8 @@
 
 import java.util.Map;
 import java.util.TreeMap;
+
+import ca.nrc.cadc.util.StringUtil;
```

Comment:
unused import, plus I know checkstyle doesn't complain much about import style, but we are trying to have ordered imports and no whitespace; can you configure your IDE for that? I'll have to make a task somewhere to improve the checkstyle... otherwise it's formatting churn when different people edit.

I didn't look at import style in  other code (diff doesn't show any issues); only noticed here because there was a new import and no new usage of that class.